ansible/roles/loadbalancer/templates/haproxy/haproxy_run.sh.j2

```diff
@@ -1,9 +1,40 @@
 #!/bin/bash -x
 
-# We need to run haproxy with one `-f` for each service, because including an
-# entire config directory was not a feature until version 1.7 of HAProxy.
-# So, append "-f $cfg" to the haproxy command for each service file.
-# This will run haproxy_cmd *exactly once*.
+{% if kolla_enable_tls_internal | bool or kolla_enable_tls_external | bool %}
+{% if kolla_enable_tls_external | bool %}
+if [ ! -e "/etc/haproxy/certificates/haproxy.pem" ]; then
```

Comment:
Would feel more comfortable if this was wrapped in an `if enable_letsencrypt`...

Reply:
https://review.opendev.org/c/openstack/kolla-ansible/+/904711

Reply:
Although I wonder how much of a corner case it is, when you have tis enabled, but don't have a cert in the container...